dotcom-rendering/src/lib/contributions.ts

```diff
@@ -220,25 +218,16 @@ export const hasCmpConsentForBrowserId = (): Promise<boolean> =>
 		});
 	});
 
-const twentyMins = 20 * 60000;
-export const withinLocalNoBannerCachePeriod = (): boolean => {
-	// eslint-disable-next-line no-restricted-syntax -- FIXME-libs-storage
-	const item = window.localStorage.getItem(NO_RR_BANNER_TIMESTAMP_KEY);
-	if (item && !Number.isNaN(parseInt(item, 10))) {
-		const withinCachePeriod = parseInt(item, 10) + twentyMins > Date.now();
-		if (!withinCachePeriod) {
-			// Expired
-			// eslint-disable-next-line no-restricted-syntax -- FIXME-libs-storage
-			window.localStorage.removeItem(NO_RR_BANNER_TIMESTAMP_KEY);
-		}
-		return withinCachePeriod;
-	}
-	return false;
-};
+export const withinLocalNoBannerCachePeriod = (): boolean =>
+	!!storage.local.get(NO_RR_BANNER_TIMESTAMP_KEY);
```

Comment:
This makes sense, but I think it's an issue for any browsers who currently have this local storage item (with no expiry). In that case I think this function will always return true, forever?
Do we need an extra PR, released before this one, to set the expiry?

Reply:
Good shout!

Reply:
I’ve now refactored this to use a new key that no longer references the timestamp. Thoughts welcome